lib/Jsrt/JsrtDebugUtils.cpp

```diff
@@ -467,7 +473,12 @@ const char16 * JsrtDebugUtils::GetClassName(Js::TypeId typeId)
     case Js::TypeIds_Promise:           return _u("Promise");
     case Js::TypeIds_GlobalObject:      return _u("Object");
     case Js::TypeIds_SpreadArgument:    return _u("Spread");
-
+#ifdef ENABLE_WASM
+    case Js::TypeIds_WebAssemblyModule:  return _u("WebAssembly.Module");
```

Comment:
Can you confirm in other browser

---
In reply to: [131437540](https://github.com/Microsoft/ChakraCore/pull/3486#discussion_r131437540) [](ancestors = 131437540)